openwisp_controller/config/models.py

```diff
@@ -71,7 +71,21 @@ class Device(OrgMixin, AbstractDevice):
     """
     Concrete Device model
     """
+    name = models.CharField(max_length=64, unique=False, db_index=True)
+    mac_address = models.CharField(
+        max_length=17,
+        db_index=True,
+        unique=False,
+        validators=[mac_address_validator],
+        help_text=_('primary mac address')
+    )
```

Comment:
why are you redefining `name` and `mac_address` here?
Aren't these inherited from `AbstractDevice`?

Reply:
I am doing this because in `AbstractDevice`, `unique=True`, but we need `unique=False`.

Reply:
@nemesisdesign